wasm/js/renderer.js

```diff
@@ -642,6 +642,9 @@ Rive.onRuntimeInitialized = function () {
         )
       );
     },
+    "rotate": function (angle) {
+      this._drawList.push(this._ctx["rotate"].bind(this._ctx, angle));
```

Comment:
This needs to update the shadow _matrixStack. Can it call transform instead?

```
const sin = Math.sin(angle);
const cos = Math.cos(radians);
this.transform({"xx": cos, "xy": sin, "yx": -sin, "yy":cos, "tx":0, "ty":0);
```

Reply:
@csmartdalton Yeah for sure! changed

Reply:
You can pull down this branch and go to `wasm/examples/centaur_game`, run `npm i`, then `npm start`. Same URL as most of the examples, you can go to `localhost:1234` and play it.